### DISCUSSION TITLE
Sbt-launch 1.5.0 not released?

### DISCUSSION THREAD

**Discussion by bblfish** [sbt/sbt#6441] [Q&A]
My [travis build is not finding](https://travis-ci.org/github/banana-rdf/banana-rdf/builds/766363425#L180) sbt-launch 1.5.0.

```log
Downloading sbt launcher for 1.5.0:
238  From  https://repo.scala-sbt.org/scalasbt/maven-releases/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar
239    To  /home/travis/.sbt/launchers/1.5.0/sbt-launch.jar
240 Downloading sbt launcher 1.5.0 md5 hash:
241  From  https://repo.scala-sbt.org/scalasbt/maven-releases/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar.md5
242    To  /home/travis/.sbt/launchers/1.5.0/sbt-launch.jar.md5
243 cat: /home/travis/.sbt/launchers/1.5.0/sbt-launch.jar.md5: No such file or directory
``` 

And indeed [it is not there](https://dl.bintray.com/sbt/maven-releases/org/scala-sbt/sbt-launch/).
Just thought I'd ping in case it is an oversight. Or perhaps I need to change something else...

**Comment by eed3si9n** [comment]
I think it's a bug in the sbt-extras that it's using (or used) `https://repo.scala-sbt.org/scalasbt/maven-releases/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar`, and it's a bug in Travis CI that it uses fake launcher instead of the official.

One way to work around it is to download the normal sbt and replace the one used on Travis CI.
